xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/Applications.xml

```diff
@@ -553,13 +550,34 @@ require(['scriptaculous/effects'], function() {
   margin-left: 0.1em;
 }
 
-.applicationPanelMoreContainer ul {
-  margin-left: 5px;
+.applicationPanelMoreButton {
+  padding: 10px 15px 10px 0;
+  border: 0;
+  color: $theme.linkColor;
+  box-shadow: none;
+  background-color: transparent;
+  width: 100%;
+  display: flex;
+}
+
+.applicationPanelMoreButton:hover {
+  background-color: $theme.highlightColor;
 }
 
-.applicationPanelMoreContainer li a {
-  font-size: 90%;
-  line-height: 16px;
+.applicationPanelMoreButton .application-label{
+  white-space: normal;
+  text-align: left;
+  display: inline;
+  margin-left: 20px;
+}
+
+.panel-width-Small .applicationPanelMoreButton {
+  flex-direction: column;
+  padding: 0;
+}
+
+.panel-width-Small .applicationPanelMoreButton .application-label{
```

Comment:
Same as above, missing space before `{`.

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/pull/2157/commits/96471a12f760ccb90ac33bfa00f3c66e98b97aef